src/atwho.js

```diff
@@ -103,7 +103,7 @@ define([
                             var match, regexp;
                             // Match text that starts with the flag and then looks like a path.
                             // CKEditor reserves the right to insert arbitrary zero-width spaces, so watch for those.
-                            regexp = new RegExp('([\\s\u200b]+|^)' + RegExp.escape(flag) + '([\\w_/]*)$', 'gi');
+                            regexp = new RegExp('([\\s\u200b]+|^)' + RegExp.escape(flag) + '([\\w-/]*)$', 'gi');
```

Comment:
`\w` doesn't seem like a valid beginning value for a range, so I'm not sure what `\w-/` will do. I think it would be safer to put the dash at the end: `([\\w/-]*)$`.

A test would be nice.